services/payments/requirements/_test.in

```diff
@@ -26,3 +26,7 @@ pytest-sugar
 python-dotenv
 python-socketio[asyncio_client]
 respx
+sqlalchemy[mypy]
+types-aiofiles # missing mypy stubs
+types-PyYAML # missing mypy stubs
+types-python-jose # missing mypy stubs
```

Comment:
just remove the comments not useful at all types and stubs it's clear what they are from the names of the libraries